Darkplaces-Emscripten/builddeps/buildjpeg.sh

```diff
@@ -0,0 +1,16 @@
+if ! type "emcc" > /dev/null; then
+    printf "\nEmscripten Not installed. Please install Emscripten at https://emscripten.org/docs/getting_started/downloads.html \n"
+    exit 1
+fi
+#doesn't work on codespaces but works on a normal computer
+git clone https://github.com/libjpeg-turbo/libjpeg-turbo .jpeg
+cd .jpeg
+emcmake cmake -G'Unix Makefiles' -DCMAKE_BUILD_TYPE=Release -DWITH_SIMD=0 .
```

Comment:
Whenever I use `emcmake cmake` it configures for **i386** instead of **x86_64**. Works fine if I use just `cmake`.

Reply:
Did you run the script? It worked on my end.

Reply:
Also, i386 is required as WASM is 32 bit

Reply:
This comment isn't really resolved because the issue appears to be [this Emscripten bug](https://github.com/emscripten-core/emscripten/issues/17268) that was supposedly fixed. 

```
$  nodejs --version
v18.19.0
 $  cmake --version
cmake version 3.28.3
```